includes/init.php

```diff
@@ -42,6 +42,8 @@
  * @package WebCalendar
  */
 
+require_once __DIR__ . '/../vendor/autoload.php';
```

Comment:
This change would be best in a separate PR.  Additionally, I prefer to not use autoload to better support offline installations and to avoid adding the vendor directory to the git repo.